ingestion/src/metadata/profiler/api/models.py

```diff
@@ -40,16 +43,40 @@ class ColumnConfig(ConfigModel):
     includeColumns: Optional[List[ColumnProfilerConfig]]
 
 
-class TableConfig(ConfigModel):
-    """table profile config"""
+class BaseProfileConfig(ConfigModel):
+    """base profile config"""
 
     fullyQualifiedName: FullyQualifiedEntityName
     profileSample: Optional[Union[float, int]] = None
     profileSampleType: Optional[ProfileSampleType] = None
+    sampleDataCount: Optional[int] = 100
+
+
+class TableConfig(BaseProfileConfig):
+    """table profile config"""
+
     profileQuery: Optional[str] = None
     partitionConfig: Optional[PartitionProfilerConfig]
     columnConfig: Optional[ColumnConfig]
 
+    @classmethod
+    def from_database_and_schema_config(
```

Comment:
nice! Let's add one unit test around this to ensure we are not missing any weird case pls

Reply:
oh sure